backend/sites/src/app/entities/siteProfiles.entity.ts

```diff
@@ -326,29 +342,29 @@ export class SiteProfiles extends ChangeAuditEntity {
   })
   govDocumentsComment: string | null;
 
-  @OneToMany(
-    () => ProfileAnswers,
-    (profileAnswers) => profileAnswers.siteProfiles,
-  )
-  profileAnswers: ProfileAnswers[];
-
-  @OneToMany(
-    () => ProfileSubmissions,
-    (profileSubmissions) => profileSubmissions.siteProfiles,
-  )
-  profileSubmissions: ProfileSubmissions[];
-
-  @OneToMany(
-    () => SiteProfileLandUses,
-    (siteProfileLandUses) => siteProfileLandUses.siteProfiles,
-  )
-  siteProfileLandUses: SiteProfileLandUses[];
-
-  @OneToMany(
-    () => SiteProfileOwners,
-    (siteProfileOwners) => siteProfileOwners.siteProfiles,
-  )
-  siteProfileOwners: SiteProfileOwners[];
+  // @OneToMany(
+  //   () => ProfileAnswers,
+  //   (profileAnswers) => profileAnswers.siteProfiles,
```

Comment:
Is this commented for future reference ? if not please remove the commented code.

Reply:
yes for future